src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        String line;
+        Task[] list = new Task[100];
+        int noOfTask = 0;
+        Scanner in = new Scanner(System.in);
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
+
         System.out.println("Hello from\n" + logo);
+
+        //Get input from user
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        line = in.nextLine();
+
+        //If bye was not inputted, ask for input again
+        while (!line.equals("bye")) {
+            //If user inputted list, return list of items
+            if (line.contains("list")) {
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < noOfTask; i++) {
+                    int listNumber = i + 1;
+                    System.out.println(listNumber + "." + list[i]);
+                }
+            //If user inputted done, then mark the task as done
+            } else if (line.contains("done")) {
```

Comment:
Instead of multiple else if, you can consider using switch case to replace it

Reply:
Thanks, will take that into consideration!